hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -172,6 +175,9 @@ public final class OmSnapshotManager implements AutoCloseable {
   // Soft limit of the snapshot cache size.
   private final int softCacheSize;
 
+  private int fsSnapshotMaxLimit;
+  private final AtomicInteger inFlightSnapshotCount = new AtomicInteger(0);
```

Comment:
I'm new to this. What does inFlightSnapshotCount count?

Reply:
`inFlightSnapshotCount` is the count of Ratis transactions that have not been applied yet.